packages/zostso/CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 All notable changes to the Zowe z/OS TSO SDK package will be documented in this file.
 
+## Recent Changes
+
+- Enhancement: Created app command under zos-tso start/send and new receive command group,
```

Comment:
Suggested changes: Issue `app` commands to better target communication with a TSO/E application. The `app` command is now included in the `start/send` command group and the new `receive` command group, allowing direct interaction with an application through a z/OS message queue.